backend/src/database/migrations/20210429045222-create-is-credential-used-function.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
       'boolean',
       'plpgsql',
       `
-    SELECT TRUE INTO result 
+    SELECT TRUE INTO result_non_email 
```

Comment:
since the original `is_credential_used` is a db function created via a migration, we would need to create a new db migration to modify it?

Reply:
I wonder what would be a good way to manage this though → ideally, we should have a single source of truths for the db functions (I have been using database browsers to look at the db function code, but I must say the current usage of db functions in the code makes tracing very difficult)

Reply:
Oh yea u're right. Nice catch

Reply:
> I wonder what would be a good way to manage this though → ideally, we should have a single source of truths for the db functions (I have been using database browsers to look at the db function code, but I must say the current usage of db functions in the code makes tracing very difficult)

Actually none of our db functions has been updated so far so I just refer to the original migration. It's not very difficult as of now as the migrations can be the single source of truth. We're planning to migrate the functions away from sql functions so let's not worry too much about this

Reply:
btw dk if you know this, but there are some db functions documented here, including the one we're modifying here: `worker/src/core/resources/sql/is-credential-used.sql`

I suspect it's dead code that we should remove?